packages/kit/src/context.ts

```diff
@@ -1,9 +1,20 @@
+import { AsyncLocalStorage } from 'node:async_hooks'
+
 import { getContext } from 'unctx'
 import type { Nuxt } from '@nuxt/schema'
 
-/** Direct access to the Nuxt context - see https://github.com/unjs/unctx. */
+/**
+ * Direct access to the Nuxt global context - see https://github.com/unjs/unctx.
+ * @deprecated Use `getNuxtCtx` instead
+ */
 export const nuxtCtx = getContext<Nuxt>('nuxt')
 
+/** async local storage for the name of the current nuxt instance */
+const asyncNameStorage = new AsyncLocalStorage<string>()
```

Comment:
We could still leverage unctx..

Reply:
cc: @huang-julien - did you try this?

Reply:
I did try but got into a `Context is not available`. I think I know what i did wrong